packages/rocketchat-ui-account/client/accountPreferences.js

```diff
@@ -56,52 +51,28 @@ Template.accountPreferences.helpers({
 		}
 		return result;
 	},
-	checked(property, value, defaultValue) {
-		const user = Meteor.user();
-
-		if (!userHasPreferences(user)) {
-			return defaultValue;
-		}
-
-		const userPreferences = user.settings.preferences;
-
-		if (userPreferences.hasOwnProperty(property)) {
-			return value === userPreferences[property];
-		}
-
-		return defaultValue;
+	checked(property, value, defaultValue=undefined) {
+		return checkedSelected(property, value, defaultValue);
 	},
-	selected(property, value, defaultValue) {
-		const user = Meteor.user();
-		const propertyeExists = !!(user && user.settings && user.settings.preferences && user.settings.preferences[property]);
-		if (propertyeExists) {
-			return user.settings.preferences[property] === value;
-		} else {
-			return defaultValue === true;
-		}
+	selected(property, value, defaultValue=undefined) {
+		return checkedSelected(property, value, defaultValue);
 	},
 	highlights() {
-		const user = Meteor.user();
-		return user && user.settings && user.settings.preferences && user.settings.preferences['highlights'] && user.settings.preferences['highlights'].join(', ');
+		const userHighlights = RocketChat.getUserPreference(Meteor.user(), 'highlights');
+		return userHighlights ? userHighlights.join(', ') : undefined;
 	},
 	desktopNotificationEnabled() {
 		return KonchatNotification.notificationStatus.get() === 'granted' || (window.Notification && Notification.permission === 'granted');
 	},
 	desktopNotificationDisabled() {
 		return KonchatNotification.notificationStatus.get() === 'denied' || (window.Notification && Notification.permission === 'denied');
 	},
-	defaultAudioNotification() {
-		return notificationLabels[RocketChat.settings.get('Audio_Notifications_Default_Alert')];
-	},
-	defaultAudioNotificationValue() {
-		return RocketChat.settings.get('Audio_Notifications_Value');
-	},
 	desktopNotificationDuration() {
-		const user = Meteor.user();
-		return user && user.settings && user.settings.preferences && user.settings.preferences.desktopNotificationDuration;
+		const userPref = RocketChat.getUserPreference(Meteor.user(), 'desktopNotificationDuration', 'undefined');
+		return userPref !== 'undefined' ? userPref : undefined;
 	},
 	defaultDesktopNotificationDuration() {
-		return RocketChat.settings.get('Desktop_Notifications_Duration');
+		return RocketChat.settings.get('Accounts_Default_User_Preferences_desktopNotificationDuration');
```

Comment:
Did you change `Desktop_Notifications_Duration` to `Accounts_Default_User_Preferences_desktopNotificationDuration` right? Is there a migration for that?

Reply:
Same for `Desktop_Notifications_Default_Alert` and `Mobile_Notifications_Default_Alert`

Reply:
Right, I forgot the migration script. :-( I Will work on it.